experiment/measurer/measure_manager.py

```diff
@@ -180,6 +198,18 @@ def save_snapshots():
     # If we have any snapshots left save them now.
     save_snapshots()
 
+    # Concatenate all process-specific data frames and remove duplicates.
```

Comment:
What does "process" mean here?

Reply:
Process was to indicate that it's specific to a single thread. Will change it to trial, as we are mapping over trials.